### PR TITLE
chore: moved driver/report.md to docs/report.md.

### PR DESCRIPTION
**Pull request by FedeDP**
Moreover, output folder will now be `libs/` instead of `driver/` since we now also change files outside of driver folder.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -192,15 +192,14 @@ func generateReport(systemMap SyscallMap) {
...
 
-	fW, err := os.Create("driver/report.md")
+	ensureFolderExists("libs/docs/")
+
+	fW, err := os.Create("libs/docs/report.md")
 	if err != nil {
 		log.Fatal(err)
 	}
 	defer fW.Close()
-	_, _ = fW.WriteString("# Supported Syscalls\n\n")
```
Comment (FedeDP) [inline main.go:200]: We now have a github page glossary for this.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**